src/main/webapp/scripts/popular.js

```diff
@@ -1,13 +1,15 @@
 import { getTrendingFromYoutubeApi } from './list-video-service.js';
 import { getRegions } from './region-selector.js';
+import { RecommendedVideoDisplay } from './classes/recommended-video-display-class.js';
+import { Video } from './classes/video-class.js';
 
 function loadPopular() {
-  return loadApi().then(popular.getTrendingFromYoutubeApi)
+  return popular.getTrendingFromYoutubeApi()
   .then((response) => {
+    const recommended = new RecommendedVideoDisplay('popular-list-container');
     for(const item of response.items) {
-      // Ready in a later PR: display the item nicely on the page
-      const containerDOM = document.getElementById('popular-list-container');
-      containerDOM.innerText += item;
+      const video = new Video(item, null);
```

Comment:
Seems odd that sentiment is not being added. Is it because it will be in another PR? Add a TODO/issue for this if so.

Reply:
Sentiment is not used for the popular videos, but it will be used for the page that displays top-sentiment videos